curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-js-fundamentals-by-building-a-gradebook-app/6626b8dcf5057f896f948440.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-4
 
 Now that the teacher has all of the information they need, they want to be able to message the student with the results. 
 
-Complete the `studentMsg` function with `totalScores` and `studentScore` for parameters. The function should return a string representing a message to the student.
+Complete the `studentMsg` function with `totalScores` and `studentScore` for parameters. The function should return string representing a message to the student.
```

Comment:
this change should not be incldued in this PR

Reply:
So what  should I do ?